beatrix/src/test/java/org/killbill/billing/beatrix/integration/TestSubscription.java

```diff
@@ -567,5 +568,65 @@ public void testCancelSubscriptionAfterTrialWith_START_OF_TERM() throws Exceptio
         assertEquals(internalCallContext.toLocalDate(subscription.getBillingEndDate()).compareTo(new LocalDate(2015, 8, 31)), 0);
 
     }
+    
+    @Test(groups = "slow")
+    public void testCreateChangeCancelSubscriptionWithDateTime() throws Exception {
+        
+    	final DateTime initialDateTime = new DateTime(2012, 5, 1, 10, 30);
```

Comment:
Isn't this `2012-05-01T10:30`? Shouldn't it be `new DateTime(2012, 5, 1, 10, 0);`?

Reply:
Yes, was a typo, I've fixed it now 👍